src/api/getArticle.js

```diff
@@ -72,6 +74,15 @@ export const getArticle = (lang, title) => {
       }
     })
 
+    // footer
+    sections.push({
+      title: i18n.i18n('toc-footer'),
```

Comment:
You can't use i18n here. The API output is transformed and cached independent of the language.

Reply:
true, and I think we have another problem here, we can change the article language, but we didn't fetch the language file for that language, is that fine the footer here "suggested articles" always the app language translation?